diagnostic.md

```diff
@@ -150,7 +166,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne is to the right, then the rest of the divs float left until there is no room and they move to the 2nd row to the left again.
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).